lib/screens/board/board_store.dart

```diff
@@ -72,10 +72,14 @@ abstract class _BoardStore with Store {
   }
 
   Future _getStatuses(BuildContext context) async {
+    statuses.clear();
     final response = await apiRequest(_api.getStatuses(), context);
     if (response.isSuccessful) {
-      statuses = ObservableList.of(response.body!);
-      statuses.sort(((a, b) => a.orderNumber.compareTo(b.orderNumber)));
+      for (var status in ObservableList.of(response.body!)) {
```

Comment:
Nie wiem w jakiej postaci przychodzą te statusy i czy są posortowane. Jeżeli nie, to trzeba je odsortować na FE, dlatego tę linię dałem

Reply:
Dodałem